### DISCUSSION TITLE
Change RDF build description output from RDF XML to TTL

### DISCUSSION THREAD

**Discussion by joegregoryphd** [opencaesar/platform#20] [General]
I see that the default RDF format output during the build task by OML Rosetta is RDF/XML – is there a way to switch the outputs to RDF/TTL? We want to be sure that we can then apply the ‘query’ task in Rosetta to RDF descriptions in the TTL format as well as the RDF XML format.

**Comment by melaasar** [comment]
The [OmlToOwl](https://github.com/opencaesar/owl-adapter/blob/master/oml2owl/README.md) tool has the param:
```
    outputFileExtension = 'owl' [Optional, default=owl, options: owl, rdf, xml, rj, ttl, n3, nt, trig, nq, trix, jsonld, fss]
```

The [OwlReason](https://github.com/opencaesar/owl-tools/blob/master/owl-reason/README.md) tool has the param:
```
	inputFileExtensions 	= ['extension'] [optional, multiple, default=owl, options: owl, rdf, xml, rj, ttl, n3, nt, trig, nq, trix, jsonld, fss]
	outputFileExtension 	= 'extension' [optional, default=ttl, options: owl, rdf, xml, rj, ttl, n3, nt, trig, nq, trix, jsonld, fss]
```
The [OwlLoad](https://github.com/opencaesar/owl-tools/blob/master/owl-load/README.md) tool has the param:
```
    fileExtensions = ['owl', 'ttl'] [Optional, default=['owl', 'ttl'], options: owl, rdf, xml, rj, ttl, n3, nt, trig, nq, trix, jsonld, fss]
```
Those should accommodate your needs for using TTL.

The OwlQuery task works with any RDF graphs loaded in a SPARQL endpoint. It's format independent.

**Reply by joegregoryphd** [reply]
Thanks Maged,
so in theory we can generate RDF descriptions (in TTL format) in other tools and load them into the SPARQL endpoint?

**Reply by melaasar** [reply]
Yes, all owl-tools, including OwlLoad and OwlQuery are not Oml-specific
